assets/js/src/core/components/tree/node/tree-node.tsx

```diff
@@ -153,8 +155,30 @@ const TreeNode = ({
     nodesRefs!.current[internalKey] = nodeRef
   }
 
+  function onDragOver (event: MouseEvent): void {
+    const assetMetaData = metaData?.asset
+
+    if (assetMetaData !== undefined && assetMetaData.type === 'folder') {
+      setSelectedIds([id])
+    }
+  }
+
+  const uploadProps: UploadProps = {
+    action: `/studio/api/assets/add/${id}`,
```

Comment:
Do we have an alternative here? 

Would be nice if we could use the normal api endpoints provided by RTK Query.
Otherwise we have to find a way to invalidate the cache tag right after the successful upload.

Reply:
no :D as we both tried it there is no alternative but we found a way :D